great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -590,13 +580,17 @@ def _build_numeric_columns_high_precision_rule() -> Rule:
                 "lower_bound": None,
                 "upper_bound": None,
             },
-            "round_decimals": 4,
+            "round_decimals": 12,
```

Comment:
This is greater than the `MAX_DECIMALS` value (9) in `NumericMetricRangeMultiBatchParameterBuilder`. Shouldn't we also increase that value and not allow it to be exceeded?